tests/common/plugins/memory_utilization/memory_utilization.py

```diff
@@ -253,7 +253,19 @@ def _handle_memory_threshold_exceeded(self, name, mem_item, value, threshold,
             logger.warning(message)
```

Comment:
@lipxu Why don't we fail the test in PR test? Then we can capture this issue in PR test rather than nightly test.